### DISCUSSION TITLE
source and source type unavailable in MP

### DISCUSSION THREAD

**Discussion by pedrozuccolo** [RADAR-base/ManagementPortal#671] [Q&A] [answered]
Hello, @nivemaham and @peyman-mohtashami, could you help us with an issue in MP? We have deployed Radar Kubernetes using MP 0.8. The MP is lacking all sources. In a previous deploy (which we have done with the same configuration only two weeks ago), everything was working properly, i.e., when we created a project in MP, we were able to select sources from a dropdown menu. Now it is empty. Also, we cannot access the list of sources in entities > sources and entities > source_types. We can create new sources and source types, but the list of already existing sources is unavailable. Is there a specific service I should install (or re-install)? We would appreciate any help. 
<img width="1280" alt="Screen Shot 2022-03-22 at 12 39 30" src="https://user-images.githubusercontent.com/94064458/159758860-87dbae1e-9bed-49e9-90cf-69b18c727e8e.png">
<img width="1280" alt="Screen Shot 2022-03-22 at 12 31 22" src="https://user-images.githubusercontent.com/94064458/159758899-d0ca5a40-69d7-4085-821e-9bba053a9bba.png">
<img width="1280" alt="Screen Shot 2022-03-22 at 12 31 15" src="https://user-images.githubusercontent.com/94064458/159758987-84395665-5045-4095-bd21-3a4d21cce826.png">

**Comment by nivemaham** [comment]
Hi @pedrozuccolo , Please check the README of the Management Portal and documentation of the helm-charts of Management Portal. You probably have to enable to catalog-server-auto-import variable to true. https://github.com/RADAR-base/ManagementPortal#environment-variables

**Comment by pedrozuccolo** [comment, marked as answer]
Hi, @nivemaham, thanks for the reply. We have checked and the MANAGEMENTPORTAL_CATALOGUE_SERVER_ENABLE_AUTO_IMPORT variable was set to true but for some reason, this service was not initiated when we deployed the system. We were able to solve the issue by deleting the pod with the catalogue service and waiting for Rancher to create another. After, we restarted the MP and then the catalog was available. Everything is working now.
